.github/workflows/integration-test.yml

```diff
@@ -14,6 +14,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ ubuntu-latest , windows-latest ]
+      number: [1,2,3,4,5,6,7,8,9,10]
```

Comment:
This change should be removed before final merge. Feel free to dismiss this review once the change is removed.